src/frontend/sider/Fagsak/Behandling/context/BehandlingContext.tsx

```diff
@@ -77,15 +82,17 @@ const BehandlingContext = createContext<BehandlingContextValue | undefined>(unde
 
 export const BehandlingProvider = ({ children }: React.PropsWithChildren) => {
     const { fagsakId } = useSakOgBehandlingParams();
-    const { hentMinimalFagsak } = useFagsakContext();
+    const queryClient = useQueryClient();
     const [åpenBehandling, privatSettÅpenBehandling] =
         useState<Ressurs<IBehandling>>(byggTomRessurs());
     const [åpenHøyremeny, settÅpenHøyremeny] = useState(true);
     const [åpenVenstremeny, settÅpenVenstremeny] = useState(true);
 
     const settÅpenBehandling = (behandling: Ressurs<IBehandling>, oppdaterMinimalFagsak = true) => {
         if (oppdaterMinimalFagsak && fagsakId) {
-            hentMinimalFagsak(fagsakId, false);
+            queryClient.invalidateQueries({
+                queryKey: HentFagsakQueryKeyFactory.fagsak(Number(fagsakId)),
```

Comment:
Hvis du bruker `useFagsakId` kan du skrive:

```suggestion
                queryKey: HentFagsakQueryKeyFactory.fagsak(fagsakId),
```